.circleci/config.yml

```diff
@@ -72,7 +72,7 @@ jobs:
       - setup_truffle_env
       - run:
           name: Run tests
-          command: truffle test 
+          command: EXPERIMENT=true npm run test
```

Comment:
what about making parametrising these jobs and running them once for`EXPERIMENT=true` and once for `EXPERIMENT=false`

Reply:
I don't think it brings any benefits now and the whole set of tests is pretty time consuming.
Currently test with `EXPERIMENT=true` runs all the test suite already.

Reply:
but, what Im worried about is that now when I run `EXPERIMENT=false` Im still getting:

```
Registering paymentToPaymentCondition (0xf204a4Ef082f5c04bB89F7D5E6568B796096735a) to spendingConditionRegistry
Registering paymentToPaymentV2Condition (0x82D50AD3C1091866E258Fd0f1a7cC9674609D254) to spendingConditionRegistry
Registering feeClaimOutputToPaymentTxCondition (0x0d8cc4b8d15D4c3eF1d70af0071376fb26B5669b) to spendingConditionRegistry
Registering paymentToPaymentV2Condition (0x1411CB266FCEd1587b0AA29E9d5a9Ef3Db64A9C5) to spendingConditionRegistry
Registering paymentV2ToPaymentV2Condition (0x855d1c79Ad3fb086D516554Dc7187E3Fdfc1C79a) to spendingConditionRegistry
Registering paymentV2ToPaymentV3Condition (0x8065F4c7b8c2bf53561af92D9DA2Ea022A0b28Ca) to spendingConditionRegistry
{"authority_address":"0xc5fdf4076b8f3a5357c5e395ab970b5b54098fef","eth_vault":"0x4d2d24899c0b115a1fce8637fca610fe02f1909e","erc20_vault":"0xc8c03647d39a96f02f6ce8999bc22493c290e734","payment_exit_game":"0xcfed223fab2a41b5a5a5f9aaae2d1e882cb6fe2d","plasma_framework_tx_hash":"0x938aa3c16a2840b79c365ffde567d1edaa0e4739402ad89ca60bb7a738e6a418","plasma_framework":"0x345ca3e014aaf5dca488057592ee47305d9b3e10"}
```

commit: `62b841c7f7673e34760567e7ebaf714c51420591`

Reply:
you mean you set `EXPERIMENT=false rpm run test` and it is still printing the log above? 😨
Or can you point me to the setup that you found this?

Reply:
So I run migrations scripts on the current HEAD and the problem does not persist (maybe that was sth with my env back then), but still one `paymentToPaymentV2Condition` is being registered (do not know if it is an issue though).

```
$ MULTI_EXIT_GAME_EXPERIMENT=true npx truffle migrate --network local | grep 'Registering.*V2'

Registering paymentToPaymentV2Condition (0x170f89CFed0bE4dd0875c8EB403094B24f7b7b34) to spendingConditionRegistry
Registering paymentToPaymentV2Condition (0x0EEa09b4E919C511Da053bD85cb88A5dD501b541) to spendingConditionRegistry
Registering paymentV2ToPaymentV2Condition (0xc29bE658676d32f16e31AC54C67875a0D1a72298) to spendingConditionRegistry
Registering paymentV2ToPaymentV3Condition (0x1f75D1F63bD0d62e56Cfa7E8B759536649DD467A) to spendingConditionRegistry
```

```
$  npx truffle migrate --network local | grep 'Registering.*V2'

Registering paymentToPaymentV2Condition (0xa5a60d16bDC4d5694b2eDF94b144e8513D4E78aE) to spendingConditionRegistry
```